.ci/deploy.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
```

Comment:
Also set two other flags 
- `-x ` to show the commands being run.
- `-o pipefail` to fail if any of the commands in a pipefail.

Reply:
Actually don't want `-x` because it'll output some private var?